src/Symfony/Component/Validator/Constraints/UniqueValidator.php

```diff
@@ -43,6 +43,7 @@ public function validate($value, Constraint $constraint)
             if (\in_array($element, $collectionElements, true)) {
                 $this->context->buildViolation($constraint->message)
                     ->setParameter('{{ value }}', $this->formatValue($value))
+                    ->setParameter('{{ duplicate_value }}', $this->formatValue($element, self::OBJECT_TO_STRING))
```

Comment:
I think I would change the name to `duplicated_element` or `repeated_element`

Reply:
I'm happy to make it whatever the consensus is, I chose this name based on the [comment on my initial bug report](https://github.com/symfony/symfony/issues/40305#issuecomment-786873635).